src/index.html

```diff
@@ -20,101 +20,112 @@
                 <a href="#" class="nav-logo"><img src="img/nav__logo.png" alt="logo"></a> 
                 <div class="nav-menu">
                     <ul class="nav-menu__list d-none d-lg-inline-block">
-                        <li class="d-inline"><a class="c-link c-link--nav h-txt9" href="#repair">Naprawa i regeneracja</a></li>
-                        <li class="d-inline"><a class="c-link c-link--nav h-txt9" href="#production">Produkcja</a></li>
-                        <li class="d-inline"><a class="c-link c-link--nav h-txt9" href="#about">O firmie</a></li>
-                        <li class="d-inline"><a class="c-link c-link--nav h-txt9 h-white" href="#contact">Kontakt</a></li>
+                        <li class="d-inline"><a class="c-link c-link--nav h-txt9 js-lang" key="nav-naprawa" href="#repair">Naprawa i regeneracja</a></li>
```

Comment:
Is `key` a valid html attribute. I used to use `data-*` for that (ref: https://www.w3schools.com/tags/att_global_data.asp)

- [ ] `data-*`